src/Pie.js

```diff
@@ -18,37 +18,81 @@ function createPath(cx, cy, r, startAngle, arcAngle) {
       r,
       startAngle,
       startAngle + arcAngle,
-    )
+    );
   } else {
-    p.path.push(0, cx + r * Math.cos(startAngle), cy + r * Math.sin(startAngle))
-    p.path.push(4, cx, cy, r, startAngle, startAngle + arcAngle, 1)
+    p.path.push(0, cx + r * Math.cos(startAngle), cy + r * Math.sin(startAngle));
+    p.path.push(4, cx, cy, r, startAngle, startAngle + arcAngle, 1);
   }
-  return p
+  return p;
 }
 
-const ArcShape = ({ radius, width, color, strokeCap,startAngle, arcAngle }) => {
+const ArcShape = ({ radius, width, color, strokeCap, startAngle, arcAngle }) => {
   const path = createPath(
     radius,
     radius,
     radius - width / 2,
     startAngle / 180 * Math.PI,
-    arcAngle / 180 * Math.PI
-  )
-  return <Shape d={path} stroke={color} strokeWidth={width} strokeCap={strokeCap} />
-}
+    arcAngle / 180 * Math.PI,
+  );
+  return <Shape d={path} stroke={color} strokeWidth={width} strokeCap={strokeCap} />;
+};
 
 const RingShape = props =>
   Platform.OS === 'ios'
     ? <ArcShape {...props} startAngle={0} arcAngle={360} />
     : <Group>
-        <ArcShape {...props} startAngle={0} arcAngle={180} />
-        <ArcShape {...props} startAngle={180} arcAngle={180} />
-      </Group>
+      <ArcShape {...props} startAngle={0} arcAngle={180} />
+      <ArcShape {...props} startAngle={180} arcAngle={180} />
```

Comment:
can we remove this workaround now?

Reply:
I'll try it and see

Reply:
Quick smoke test tells me this can be removed.

What was the original need for this? We should test that particular scenario to prove this is not needed